plugins/rtmp-services/data/services.json

```diff
@@ -1152,16 +1152,12 @@
         {
             "name": "Switchboard Live",
             "alt_names": [
-                "Switchboard Live (Joicaster)"
+                "Switchboard Live"
```

Comment:
The previous comment still has not been addressed, see https://github.com/obsproject/obs-studio/pull/10021#discussion_r1431835434.

Reply:
@sarwin-github This has still not been addressed. Do not change existing entries in alt_names. They are there for backwards compatibility reasons. We will not merge this as-is.

Reply:
I fixed this manually.